org.jacoco.core/src/org/jacoco/core/internal/flow/Instruction.java

```diff
@@ -102,11 +121,11 @@ public int getBranches() {
 	}
 
 	/**
-	 * Returns the number of covered branches starting from this instruction.
-	 * 
-	 * @return number of covered branches
+	 * Returns coverage of branches of this instruction.
+	 *
+	 * @return coverage of branches of this instruction
 	 */
-	public int getCoveredBranches() {
+	public BitSet getCoveredBranches() {
```

Comment:
@marchof encapsulated, added `toString` instead of package-local field to avoid its accidental usage in other classes